src/test/java/helper/Base.java

```diff
@@ -0,0 +1,17 @@
+package helper;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class Base {
```

Comment:
Maybe we can have a better class name. Smth like ,,Initialization''

Reply:
Fixed